.gitignore

```diff
@@ -12,6 +12,7 @@ build/*
 zulip_term.egg-info
 zulip-terminal-tracebacks.log
 zulip-terminal-API-requests.log
+zulip-terminal-thread-exceptions.log
```

Comment:
Given the common naming we have for log files, this might be simpler covering all logfiles with one pattern?

Reply:
That makes sense. Since we exclude all logfiles from being pushed, I can think of `*.logs`. Any logfiles being added in the future would also be excluded.

Reply:
That makes sense. Since we exclude all logfiles from being pushed, I can think of `*.log`. Any logfiles being added in the future would also be excluded.

Reply:
I was originally thinking of `zulip-terminal-*.log`, but that might be cleaner :+1:

We could better split the gitignore into sections and maybe add comments, but it's not that large and time is likely better spent elsewhere :)